cmd/main.go

```diff
@@ -1040,7 +1014,7 @@ func executeBlobCmd(meth method, pth string, pars params) error {
 
 func request(meth method, pth string, pars params) (*http.Response, func(), error) {
 	apiURL := fmt.Sprintf("%s/api/%s/%s", *apiAddr, *apiVersion, pth)
-	if *logDebug {
+	if *daemonLogDebug {
```

Comment:
Ok, I should have documented this change.

Before `--debug` existed for daemon command, and init command. I decided to make it a global flag instead, which is common for CLIs, hence this functionality which operates on the CLI client - so doing `textile files get <hash>` will output the request URL that it made.

This functionality will need to be removed, or `--debug` will need to be re-added as a global flag instead.

Reply:
Ah, nice. I like that. Will change back.